README.md

```diff
@@ -89,6 +91,11 @@ persist in Tigerblood while the process executes. Configuration for `file` is ju
 The `aws` exception module adds known AWS public IP subnets to the exception list, and are polled periodically. The `aws`
 module has no configuration options, and can be invoked by specifying `aws=` with no configuration parameter.
 
+## Authentication
+
+Tigerblood will authenticate requests using either Hawk style authentication, static API keys, or both. The `HAWK` and `APIKEY` configuration values can be used to control this behavior. With static API key authentication, the
```

Comment:
Should we say "either of them" instead of "both"?

To me, "both" means the client needs a valid hawk header and API key, which I don't think the code is doing.